src/components/CallView/shared/LocalMediaControls.vue

```diff
@@ -82,6 +81,42 @@
 					</li>
 				</ul>
 			</div>
+			<button
+				v-shortkey="['h']"
```

Comment:
Jitsi uses `r` for `raise hand`.
Can someone check teams? If they also use `r` we should use `r` so we are inline with expectations of users.

Reply:
I've changed it to "r" for now. we can change it later once someone can check on teams